<ui version="4.0" >
 <class>vvDicomSeriesSelector</class>
 <widget class="QDialog" name="vvDicomSeriesSelector" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>645</height>
   </rect>
  </property>
  <property name="mouseTracking" >
   <bool>true</bool>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Dicom Series Selector</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QFrame" name="frame" >
      <property name="frameShape" >
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth" >
       <number>1</number>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QSplitter" name="splitter" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget0" >
          <layout class="QVBoxLayout" name="verticalLayout" >
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout" >
             <item>
              <widget class="QLabel" name="label_2" >
               <property name="text" >
                <string>Search in this folder :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="mFolderLineEdit" >
               <property name="frame" >
                <bool>false</bool>
               </property>
               <property name="dragEnabled" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="mBrowseButton" >
               <property name="text" >
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" >
             <item>
              <widget class="QCheckBox" name="mIsRecursiveCheckBox" >
               <property name="text" >
                <string>Search recursively</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="mSearchButton" >
               <property name="text" >
                <string>Search DICOM</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="mListWidget" >
             <property name="mouseTracking" >
              <bool>true</bool>
             </property>
             <property name="focusPolicy" >
              <enum>Qt::TabFocus</enum>
             </property>
             <property name="editTriggers" >
              <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
             </property>
             <property name="tabKeyNavigation" >
              <bool>true</bool>
             </property>
             <property name="alternatingRowColors" >
              <bool>true</bool>
             </property>
             <property name="selectionBehavior" >
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="viewMode" >
              <enum>QListView::ListMode</enum>
             </property>
             <property name="modelColumn" >
              <number>0</number>
             </property>
             <property name="currentRow" >
              <number>-1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="1" >
        <layout class="QVBoxLayout" name="_2" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="mLabelSelected" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>&lt;b>Current DICOM serie&lt;/b></string>
           </property>
           <property name="textFormat" >
            <enum>Qt::RichText</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="mDicomInfoPanel" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="textFormat" >
            <enum>Qt::RichText</enum>
           </property>
           <property name="alignment" >
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_2" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QSplitter" name="splitter_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget1" >
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="mLabelSelected_3" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>&lt;b>Files in current DICOM serie&lt;/b></string>
             </property>
             <property name="textFormat" >
              <enum>Qt::RichText</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="mDicomDetailsListWidget" >
             <property name="focusPolicy" >
              <enum>Qt::TabFocus</enum>
             </property>
             <property name="alternatingRowColors" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget2" >
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="mLabelSelected_2" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>&lt;b>Current DICOM file&lt;/b></string>
             </property>
             <property name="textFormat" >
              <enum>Qt::RichText</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="mDicomDetailsLabel" />
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>&lt;font color="blue">(1)&lt;/font> Select a folder,  &lt;font color="blue">(2)&lt;/font> Press &lt;b>Search DICOM&lt;/b>, &lt;br> &lt;font color="blue">(3)&lt;/font> Select a serie ID in the left table, &lt;font color="blue">(4)&lt;/font> Press &lt;b>Open&lt;/b></string>
       </property>
       <property name="textFormat" >
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>141</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="mButtonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mDicomDetailsListWidget</tabstop>
  <tabstop>mButtonBox</tabstop>
  <tabstop>mDicomDetailsLabel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>vvDicomSeriesSelector</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>vvDicomSeriesSelector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
